packages/ethereum-contracts/contracts/mocks/SuperAppBaseFlowMock.sol

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity >= 0.8.0;
+
+import {ISuperfluid, ISuperToken, ISuperApp, SuperAppDefinitions} from "../interfaces/superfluid/ISuperfluid.sol";
+import {SuperTokenV1Library} from "../apps/SuperTokenV1Library.sol";
+
+abstract contract SuperAppBaseFlowMock is ISuperApp {
+    using SuperTokenV1Library for ISuperToken;
+    
+    ISuperfluid public _host;
+
+    /// @dev Can be used to track accepted status of tokens for Super App
+    mapping(ISuperToken => bool) internal _acceptedSuperTokens;
+
+    bytes public beforeCtxHolder;
+    bytes public afterCtxHolder;
+    string public callbackHolder;
```

Comment:
to be checked: what exactly are you testing here? Is this really about the SuperAppBaseFlow or redundantly checking the underlying mechanisms?

Reply:
It may be redundant checking, but definitely zooming into underlying behavior that happens during calls and doesn't reflect in post-state